Add B11 Implementation

TrueFinch opened the pull request:
https://www.notion.so/EGE-B11-731cdf403b7c492e81557bc0906b77b7 — Генератор B11

TrueFinch committed:
Add IpMask task of B11

https://www.notion.so/EGE-B11-731cdf403b7c492e81557bc0906b77b7 — Генератор B11 IpMask - указать адрес подсети по IP-адресу и маске подсети.

TrueFinch committed:
Add SubnetMask task of B11

https://www.notion.so/EGE-B11-731cdf403b7c492e81557bc0906b77b7 — Генератор B11 subnet_mask - найти наибольший (или наименьший) возможный или конкретный третий байт маски по заданным адресам ip и сети
Fix uncertainty in the condition of the task: in case ip=235.45.82.249 and subnet=235.45.64.0. Old generator gives answer 224, but according condition 192 was right too

TrueFinch commented:
![изображение](https://user-images.githubusercontent.com/24905983/150415918-4d1895ea-e99d-4aef-81dc-2b4b1451fd5d.png)

Собственно формулировка на которой ответ некорректен. По условию просят указать конкретное значение третьего байта маски. Возможны два значения 224 и 192. Генератор учитывал только наибольшее. Поэтому я слегка переписал условие и генератор. Теперь есть три варианта постановки вопроса, под которые подстраивается генератор, подбирая числа так, чтобы неопределённости не возникало

tonichoff commented on EGE/Gen/EGE/B11.py:
```diff
@@ -0,0 +1,139 @@
+        return (
+            ''.join([ header[i] for i in map(find_part, ip) ]),
+            html.table(html.row('th', header) + html.row('td', parts), border=1)
+        )
```
Нет пустой строки между методами

tonichoff commented on EGE/Gen/EGE/B11.py:
```diff
@@ -0,0 +1,139 @@
+Для узла с IP-адресом {ip_address} адрес сети равен {subnet_network} . {question} Ответ запишите в виде десятичного числа."""
+        self.correct = answer
+        return self
+
```
Лишняя пустая строка

TrueFinch committed:
Fix code style

https://www.notion.so/EGE-B11-731cdf403b7c492e81557bc0906b77b7 — according to
https://github.com/klenin/EGEpy/pull/45#discussion_r789263731
https://github.com/klenin/EGEpy/pull/45#discussion_r789263247
